packages/svelte/src/internal/client/dom/elements/bindings/this.js

```diff
@@ -10,7 +10,7 @@ import { queue_micro_task } from '../../task.js';
  */
 function is_bound_this(bound_value, element_or_component) {
 	// Find the original target if the value is proxied.
-	var proxy_target = bound_value && bound_value[STATE_SYMBOL]?.t;
+	var proxy_target = bound_value && proxies.get(bound_value)?.t;
```

Comment:
A bit unrelated: Will this logic even work at all now? If it gets the underlying value, which IIUC is the original object, that one will likely not be the current value because of the change